sdk/dotnet/StorageGateway/Inputs/NfsFileShareNfsFileShareDefaultsArgs.cs

```diff
@@ -28,13 +28,13 @@ public sealed class NfsFileShareNfsFileShareDefaultsArgs : Pulumi.ResourceArgs
         /// The default group ID for the file share (unless the files have another group ID specified). Defaults to `65534` (`nfsnobody`). Valid values: `0` through `4294967294`.
         /// </summary>
         [Input("groupId")]
-        public Input<int>? GroupId { get; set; }
+        public Input<string>? GroupId { get; set; }
 
         /// <summary>
         /// The default owner ID for the file share (unless the files have another owner ID specified). Defaults to `65534` (`nfsnobody`). Valid values: `0` through `4294967294`.
         /// </summary>
         [Input("ownerId")]
-        public Input<int>? OwnerId { get; set; }
+        public Input<string>? OwnerId { get; set; }
```

Comment:
Breaking changes - let's make sure they are okay

Reply:
I am going to apply the rule here that it's ok to adopt these changes as this will affect a small number of people and will not have any destructive behaviour on the resources - it is just a type change in their code